src/test/java/no/sikt/nva/scrapers/EntityDescriptionExtractorTest.java

```diff
@@ -27,27 +29,52 @@ void shouldMapLocalCodeToDescription() {
-                                                      LOOKUP_IN_CHANNEL_REGISTER,
-                                                      Map.of());
-        var record = dublinCoreScraper.validateAndParseDublinCore(dublinCore,
-                                                                  new BrageLocation(null),
-                                                                  SOME_CUSTOMER);
+        var dublinCoreScraper = new DublinCoreScraper(ONLINE_VALIDATION_DISABLED, LOOKUP_IN_CHANNEL_REGISTER, Map.of());
+        var record = dublinCoreScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(null), SOME_CUSTOMER);
         var descriptions = record.getEntityDescription().getDescriptions();
         assertThat(descriptions, hasItem(someLocalCode));
     }
 
     @Test
     void shouldKeepOriginalOrderOfDescriptions() {
-        var dublinCore = DublinCoreFactory.createDublinCoreFromXml(new File(TEST_RESOURCE_PATH +
-                                                                            "dc_with_duplicated_values.xml"));
-        var dublinCoreScraper = new DublinCoreScraper(ONLINE_VALIDATION_DISABLED,
-                                                      LOOKUP_IN_CHANNEL_REGISTER,
-                                                      Map.of());
-        var record = dublinCoreScraper.validateAndParseDublinCore(dublinCore,
-                                                                  new BrageLocation(null),
-                                                                  SOME_CUSTOMER);
+        var dublinCore = DublinCoreFactory.createDublinCoreFromXml(
+            new File(TEST_RESOURCE_PATH + "dc_with_duplicated_values.xml"));
+        var dublinCoreScraper = new DublinCoreScraper(ONLINE_VALIDATION_DISABLED, LOOKUP_IN_CHANNEL_REGISTER, Map.of());
+        var record = dublinCoreScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(null), SOME_CUSTOMER);
         var descriptions = record.getEntityDescription().getDescriptions();
         assertThat(descriptions, is(equalTo(List.of("First description", "Second description"))));
     }
+
+    @Test
+    void shouldCreateContributorWithOrcidWhenBrageRecordHasSingleContributorAndSingleOrcId() {
+        var orcId = randomString();
+        var dublinCore = DublinCoreFactory.createDublinCoreWithDcValues(
+            List.of(randomContributor(), orcIdWithValue(orcId)));
+        var dublinCoreScraper = new DublinCoreScraper(ONLINE_VALIDATION_DISABLED, LOOKUP_IN_CHANNEL_REGISTER, Map.of());
+        var record = dublinCoreScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(null), SOME_CUSTOMER);
+        var contributor = record.getEntityDescription().getContributors().get(0);
+
+        assertThat(contributor.getIdentity().getOrcId(), is(equalTo(orcId)));
+    }
+
+    @Test
+    void shouldCreateContributorWithoutOrcidWhenBrageRecordHasMultipleContributorsAndOrcIds() {
```

Comment:
Might do a parameterised test for e.g. mutliple orcids and single contributor, single orcid and multiple contributors, multiple orcids and multiple contributors. (Sounds ridiculous, but we know the data probably contains any of these combinations)

Reply:
Adding parameterized test.